xprof/btx_timeline.cpp

```diff
@@ -72,78 +78,122 @@ static perfetto_uuid_t get_parent_counter_track_uuid(timeline_dispatch_t *dispat
   auto *process = track_descriptor->mutable_process();
   process->set_pid(hp_uuid);
   std::ostringstream oss;
-  oss << "Hostname " << hostname << " | Process " << process_id << " | Device " << did;
+  oss << "Hostname " << hostname << " | Device " << deviceIdx;
```

Comment:
Do we need that?
I would like this PR to not change the default non sampling timeline.

Reply:
This is not non sampling timeline. We were printing process_id but it is always 0 for sampling. The non sampling function still uses process_id and safe :) .

Reply:
What do you mean `get_parent_counter_track_uuid` is not used by the regular timeline? Only by the sampling?

Reply:
Oh good call. Missed the `counter` in the name! Thanks